### ISSUE TITLE
feat: In the background of IM group list management, the fuzzy query of group names is supported

### ISSUE THREAD

**Issue by cubxxw** [openimsdk/open-im-server#2099]
![image](https://github.com/openimsdk/open-im-server/assets/86140903/110f9822-e1ef-4f3e-9bdb-c7e7643a726f)
![image](https://github.com/openimsdk/open-im-server/assets/86140903/fe4582e0-6c84-4301-b2b6-95a066581309)

**Comment by cubxxw** [comment]
api: get_groups

**Comment by kubbot** [comment]
This issue is available for anyone to work on. **Make sure to reference this issue in your pull request.** :sparkles: Thank you for your contribution! :sparkles:
[Join slack 🤖](https://join.slack.com/t/openimsdk/shared_invite/zt-22720d66b-o_FvKxMTGXtcnnnHiMqe9Q) to connect and communicate with our developers.
If you wish to accept this assignment, please leave a comment in the comments section: `/accept`.🎯

**Comment by Lansongxx** [comment]
I noticed that there seems to be a fuzzy query for group names already available
![图片](https://github.com/openimsdk/open-im-server/assets/62054128/cf6c55b1-c37c-488e-bbe9-4b0cfd558096)

**Comment by cubxxw** [comment]
@Lansongxx Have you deployed openim locally, including the admin backend?

**Comment by Lansongxx** [comment]
@cubxxw    Do you mean you need to write front-end code for the backend? I'm probably not very good at front-end.

**Comment by cubxxw** [comment]
I'm not sure if this issue has been fixed in the new refactoring.

One of the first problems I found was that we had to replace mysql with mongo halfway through.
After we finished the replacement work, we tried to test the admin background, but found that there would be some issues with the interface.

It's easy to see this problem in our administrative backend system after deployment with `docker compose up -d`.
